types/decimal.go

```diff
@@ -319,7 +319,7 @@ func (d Dec) QuoRoundUp(d2 Dec) Dec {
 	quo := new(big.Int).Quo(mul, d2.i)
 	chopped := chopPrecisionAndRoundUp(quo)
 
-	if chopped.BitLen() > 255+DecimalPrecisionBits {
+	if chopped.BitLen() > (maxBitLen + DecimalPrecisionBits) {
```

Comment:
Let's create a constant for `maxBitLen + DecimalPrecisionBits`. It's repeated few times.

Reply:
fixed